internal/function/language.go

```diff
@@ -161,6 +161,22 @@ func (f *Language) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 	return lang, nil
 }
 
+func getLanguage(filename string, blob []byte) string {
+	if l := enry.GetLanguage(filename, blob); l != "" {
+		return l
+	}
+	if l, _ := enry.GetLanguageByContent(filename, blob); l != "" {
+		return l
+	}
+	if l, _ := enry.GetLanguageByFilename(filename); l != "" {
+		return l
+	}
+	if l, _ := enry.GetLanguageByExtension(filename); l != "" {
+		return l
+	}
+	return ""
+}
```

Comment:
This is already done by `enry.GetLanguage` as far as I know.

`enry.GetLanguage` calls `enry.GetLanguages` (https://github.com/src-d/enry/blob/master/common.go#L121) which calls all these strategies: https://github.com/src-d/enry/blob/master/common.go#L20-L27

Reply:
ok, so I'll change it and just call `GetLanguages`, because so far `GetLanguage` doesn't check if returned string is empty.
My intention was to add support for `.h` (c header files) which is missed.

Reply:
There is a bug in enry: https://github.com/src-d/enry/blob/master/common.go#L139
It populates a list of candidates but returns an empty slice of languages.